### PR TITLE
Add mandat creation temporary form

### PR DESCRIPTION
**Pull request by thimy**
- create a form where a user can choose a Demarche
- links to a fake FC page
- resume form and make the usager validate it

### CODE REVIEW

File: aidants_connect_web/views.py [hunk 1]
```diff
@@ -2,19 +2,31 @@
...
 from django.views.decorators.csrf import csrf_exempt
 from django.shortcuts import render, redirect
+from django.contrib.auth import logout
 from django.contrib.auth.decorators import login_required
 from django.core.exceptions import ObjectDoesNotExist
 from django.forms.models import model_to_dict
 from django.conf import settings
 from django.utils import timezone
+from django.contrib.sessions.backends.db import SessionStore
+from django.core import serializers
```
Comment (Anna-Livia) [inline aidants_connect_web/views.py]: Is this still used ?

File: aidants_connect_web/views.py [hunk 2]
```diff
@@ -28,6 +40,146 @@ def home_page(request):
     )
 
 
+@login_required
+def logout_page(request):
```
Comment (Anna-Livia) [inline aidants_connect_web/views.py]: Great feature ;)

File: aidants_connect_web/views.py [hunk 3]
```diff
@@ -28,6 +40,146 @@ def home_page(request):
...
+    logout(request)
+    return redirect(settings.LOGOUT_REDIRECT_URL)
+
+
+@login_required
+def dashboard(request):
+    messages = get_messages(request)
+    user = User.objects.get(id=request.user.id)
```
Comment (Anna-Livia) [inline aidants_connect_web/views.py]: There might be a better way to get the current user

File: aidants_connect_web/views.py [hunk 4]
```diff
@@ -28,6 +40,146 @@ def home_page(request):
...
+    mandats = Mandat.objects.all().filter(aidant=request.user)
```
Comment (Anna-Livia) [inline aidants_connect_web/views.py]: You can set the display order here

File: aidants_connect_web/views.py [hunk 5]
```diff
@@ -28,6 +40,146 @@ def home_page(request):
...
+        request,
+        "aidants_connect_web/dashboard.html",
+        {"user": user, "mandats": mandats, "messages": messages},
+    )
+
+
+@login_required
+def mandat(request):
+
+    user = User.objects.get(id=request.user.id)
```
Comment (Anna-Livia) [inline aidants_connect_web/views.py]: See above comment

File: aidants_connect_web/admin.py [hunk 6]
```diff
@@ -1,5 +1,7 @@
 from django.contrib import admin
 from django.contrib.auth.admin import UserAdmin
-from .models import User
+from .models import User, Usager, Mandat
 
 admin.site.register(User, UserAdmin)
+admin.site.register(Usager)
```
[the remark this answers is not in the document]
Comment (thimy) [reply aidants_connect_web/admin.py]: I just used it to check on the db and to let Noemie and Berangere have a glimpse on it.

**Pull request merged**